sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Properties/AssemblyInfo.cs

```diff
@@ -4,8 +4,8 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyTitle("Microsoft Azure KubernetesConfiguration Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure KubernetesConfiguration management operations including the ability to create, update and delete runbooks and schedules.")]
+[assembly: AssemblyTitle("Microsoft Azure KubernetesConfiguration SourceControlConfigurations Management Library")]
+[assembly: AssemblyDescription("Provides Microsoft Azure KubernetesConfiguration SourceControlConfigurations management operations including the ability to create, update and delete configurations.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
I want to double confirm with you that it is intentional change to split the namespace `Microsoft.Azure.Management.KubernetesConfiguration` into `Microsoft.Azure.Management.KubernetesConfiguration.SourceControlConfigurations` and `Microsoft.Azure.Management.KubernetesConfiguration.Extensions`. If so, you need to bump the major version here.